src/main/java/cubicchunks/CubicChunks.java

```diff
@@ -78,4 +92,60 @@ public void init(FMLInitializationEvent event) {
 	public void onServerAboutToStart(FMLServerAboutToStartEvent event) {
 		proxy.setBuildLimit(event.getServer());
 	}
+
+	@SubscribeEvent
+	public void onConfigChanged(ConfigChangedEvent.OnConfigChangedEvent eventArgs) {
+		if(eventArgs.getModID().equals(CubicChunks.MODID))
+			Config.syncConfig();
+	}
+
+	public static class Config {
+		private static int maxGeneratedCubesPerTick;
+		private static int maxTimeSpentGeneratingCubesPerTick;
+		private static int verticalCubeLoadDistance;
+		private static int worldHeightLowerBound;
+		private static int worldHeightUpperBound;
+		private static Configuration configuration;
+
+		static void loadConfig(Configuration configuration) {
+			Config.configuration = configuration;
+		}
+
+		static void syncConfig() {
+			maxGeneratedCubesPerTick = configuration.getInt("maxGeneratedCubesPerTick", Configuration.CATEGORY_GENERAL,
+					49*16, 1, Integer.MAX_VALUE, "The number of cubic chunks to generate per tick.");
+			maxTimeSpentGeneratingCubesPerTick = configuration.getInt("maxTimeSpentGeneratingCubesPerTick", Configuration.CATEGORY_GENERAL, 50, 1, Integer.MAX_VALUE, "The maximum amount of time in milliseconds per tick to spend generating chunks.");
+			verticalCubeLoadDistance = configuration.getInt("verticalCubeLoadDistance", Configuration.CATEGORY_GENERAL, 8, 2, 32, "Similar to Minecraft's view distance, only for vertical chunks.");
+			worldHeightLowerBound = configuration.getInt("worldHeightLowerBound", Configuration.CATEGORY_GENERAL, -4096, -8388608, 0, "The lower boundary on the world. Blocks will not generate or load below this point.");
+			worldHeightUpperBound = configuration.getInt("worldHeightUpperBound", Configuration.CATEGORY_GENERAL, 4096, 256, 8388608, "The upper boundary on the world. Blocks will not generate or load above this point.");
```

Comment:
Replace height constants with AddressTools.MIN_BLOCK_Y and AddressToold.MAX_BLOCK_Y